sdk/spring/tests.yml

```diff
@@ -1,9 +1,33 @@
 trigger: none
 
 jobs:
-  - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
-    parameters:
-      ServiceDirectory: spring
-      EnvVars:
-        AZURE_TEST_MODE: LIVE
-      Artifacts: []
+    - template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+      parameters:
+        TimeoutInMinutes: 240
+        MaxParallel: 2
```

Comment:
`MaxParallel` is probably not necessary here. For the most part you probably only need to set it when one or more of these is true: 

* Tests use some shared resource (you're using `test-resources.json` which gives you one set of resources per job) 
* The resources you're creating have a low upper bound for the number of instances which can be created in a subscription and you want to limit parallelism to prevent having too many instances of that resource active at once
* Your test matrix is enormous and running tests would saturate our agent pool and make other tests wait a long time in a queued state

Reply:
Okay, I'll remove it then.